components/gmLogger.js

```diff
@@ -18,7 +18,7 @@ function gmLogger() {
 
   // Get the current debug preference value (silent)
   //this._debug = this._branch.getBoolPref("debug");
-  this._debug = true;
+  this._debug = false;
```

Comment:
Will use commented block instead, there is no need to have log turned on in released instances.